README.md

```diff
@@ -1,25 +1,13 @@
 # Bluedot-React-Native-Minimal-Integration
 
 This App is a minimal App depicting integration of Bluedot Plugin(bluedot-react-native) https://www.npmjs.com/package/bluedot-react-native
-with a React Native App supporting react native version 0.61.2
-
-Additional steps to follow while building for Android App as give build error for AndroidX:
-
-In below path, in the dependencies section change "compileOnly" to "implementation" for 
-/Users/UserName/react-native-minimal-integration/node_modules/@unimodules/react-native-adapter/android/build.gradle
-
-
-### Old:
-compileOnly('com.facebook.react:react-native:+') {
-    exclude group: 'com.android.support'
-}
-  
-### New:
-implementation('com.facebook.react:react-native:+') {
-    exclude group: 'com.android.support'
-}
-
-More Details on above issue can be found at:
-
-https://github.com/react-native-community/react-native-maps/issues/2940
+with a React Native App supporting react native version 0.67.3
+
+### Notes:
+If you encounter issues while running the react native apps on iOS and Android, try these steps:
+1. Clear watchman watches: `watchman watch-del-all`
+2. Delete node_modules and package-lock.json and reinstall packages: `rm -rf node_modules && rm -rf package-lock.json` 
+3. Reinstall packages: `npm install` 
+4. Start fresh by resetting cache: `npm start --reset-cache`
+5. Run again: `expo run:ios` or `expo run:android`
```

Comment:
Sometimes I run into this cache issue, have some guide here to fix that. Let me know if this issue happens to you as well.